service/controller/resource/pvc/delete.go

```diff
@@ -77,13 +77,13 @@ func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange inte
 }
 
 func (r *Resource) NewDeletePatch(ctx context.Context, obj, currentState, desiredState interface{}) (*crud.Patch, error) {
-	delete, err := r.newDeleteChange(ctx, obj, currentState, desiredState)
+	deleteChange, err := r.newDeleteChange(ctx, obj, currentState, desiredState)
 	if err != nil {
 		return nil, microerror.Mask(err)
 	}
 
 	patch := crud.NewPatch()
-	patch.SetDeleteChange(delete)
+	patch.SetDeleteChange(deleteChange)
```

Comment:
I dont get this part, why you do a patch to delete a PVC?

Reply:
In our `operatorkit` CRUD resource handlers, a patch includes create, update, and delete steps. So we're not patching the PVC (in the kubernetes sense). Instead, the patch that we're applying is to delete them. Maybe the naming could be improved here, but this particular change was to avoid an editor warning that `delete` is a reserved word in Go.